aten/src/ATen/native/transformers/cuda/sdp_utils.cpp

```diff
@@ -412,7 +412,13 @@ bool check_cudnn_tensor_shapes(sdp_params const& params, bool debug) {
     }
     return false;
   }
-  constexpr auto head_dim_limit = 128;
+  auto head_dim_limit = 128;
+  if (cudnn_version >= 90501) {
+    auto dprops = at::cuda::getCurrentDeviceProperties();
+    if (dprops->major >= 9 && !dprops->minor) {
```

Comment:
So Blackwell doesn't support less than 256? What about consumer GPU Blackwell? Same story?

Reply:
Also just want to confirm cudnn_frontend doesn't give any way to query device support for head dim?